### ISSUE TITLE
Calling unknown method: nystudio107\instantanalytics\InstantAnalytics::iaSendPageView()

### ISSUE THREAD

**Issue by bryantAXS** [nystudio107/craft3-instantanalytics#2]
After a Craft 3 and Commerce 2 migration.

When I comment out

```
{% block analytics %}
    {% hook 'iaSendPageView' %}
  {% endblock %}
```

in the templates, the error goes away.

```
yii\base\UnknownMethodException: Calling unknown method: nystudio107\instantanalytics\InstantAnalytics::iaSendPageView() in /home/vagrant/Code/clients/safebonesco/website-craft3/vendor/yiisoft/yii2/base/Component.php:300
Stack trace:
#0 /home/vagrant/Code/clients/safebonesco/website-craft3/vendor/craftcms/cms/src/web/View.php(1225): yii\base\Component->__call('iaSendPageView', Array)
#1 /home/vagrant/Code/clients/safebonesco/website-craft3/storage/runtime/compiled_templates/4c/4cfa2cbc4a82ead844fa6cb29781beec9274bab65c1bad565b7ee62a0651bcfc.php(373): craft\web\View->invokeHook('iaSendPageView', Array)
```

**Comment by khalwat** [comment]
Sorry, I don't know how I missed this issue. This should be fixed now yes?

**Comment by khalwat** [comment]
Fixed in https://github.com/nystudio107/craft-instantanalytics/commit/0b4ab18e40ce3fc5d667e84eaa053c08c15cb370

This fix is rolled into Instant Analytics 1.0.8
